include/sqlpp11/sqlite3/connection_config.h

```diff
@@ -37,14 +37,21 @@ namespace sqlpp
 	{
 		struct connection_config
 		{
-			std::string path_to_database;
-			int flags = 0;
-			std::string vfs = "";
-			bool debug = false;
+			connection_config() = default;
```

Comment:
This will leave the member variables uninitialized. Can you add default initialization to the constructor?